shader/f.glsl

```diff
@@ -0,0 +1,5 @@
+uniform vec4 color;
```

Comment:
Similarily to the last comment, shouldn't the shaders be contained in the submodule?

Reply:
I thought the shader folder were gone, but it isn't used anymore